content/20.treatments.md

```diff
@@ -472,13 +482,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
 
 ###### Current Evidence
 
-A recent preprint postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19.
-This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations [@doi:10.20944/preprints202003.0235.v2].
+A recent review postulated that an individual’s vitamin D status may significantly affect their risk of developing COVID-19.
```

Comment:
I would stick to the Calcifediol/Calcitriol or 25-OH/1,25-OH terminology and introduce them here because they are short and comprehensible. Where is the preprint reference?